dbt_platform_helper/commands/dns.py

```diff
@@ -584,7 +584,12 @@ def assign(app, domain_profile, project_profile, svc, env):
     domain_session = get_aws_session_or_abort(domain_profile)
     project_session = get_aws_session_or_abort(project_profile)
 
-    ensure_cwd_is_repo_root()
+    if not Path("./copilot").exists() or not Path("./copilot").is_dir():
+        click.secho(
+            "Cannot find copilot directory. Run this command in the root of the deployment repository.",
+            bg="red",
+        )
+        exit(1)
```

Comment:
Why are we not just using [ ensure_cwd_is_repo_root()](https://github.com/uktrade/platform-tools/blob/main/dbt_platform_helper/utils/files.py#L38)?

Reply:
It was only being used in 2 places, one of which was doing a check_config() anyway which will error if not in the root directory. So this is the only place ensure_cwd_is_root is used so I inlined it (so people don't continue using it). Checking you are in the same dir as the platfom-config.yml file will work going forward when we eventually free ourselves from copilot (or if the copilot dirs are not there as we haven't generated them yet)

I didn't just use check_config() here as I don't think the command actually needs the config file.